web-portal/package.json

```diff
@@ -39,7 +39,9 @@
     "nuxt-clipboard2": "^0.2.1",
     "slack-notify": "^2.0.2",
     "uuid": "^9.0.0",
+    "v-calendar": "^2.4.2",
     "vue-clipboard2": "^0.3.0",
+    "vue-flatpickr-component": "^8.1.8",
```

Comment:
Is `v-calendar` required w/ flatpickr?

Reply:
hope not, I think I left that in there from my earlier experiments

Reply:
516fdb169070de85969595c2c65016f684e56999